setup.py

```diff
@@ -69,4 +70,19 @@ def run(self):
     packages=["ucp"],
     ext_modules=ext_modules,
     cmdclass={"build_ext": build_ext},
```

Comment:
We need to also include the command classes from versioneer. They don't include `build_ext` (last I checked). So it won't overwrite that one.

```suggestion
     cmdclass={"build_ext": build_ext, **versioner.get_cmdclass()},
```